static-assets/components/cstudio-admin/mods/admin-configurations.js

```diff
@@ -114,7 +114,15 @@
           /****/'<i class="hint-text--icon fa fa-info" aria-hidden="true"></i>' +
           /****/'<div class="hint">' +
           /******/`<h2 class="hint--title">${formatMessage(adminConfigurationMessages.encryptMarked)}</h2>` +
-          /******/`<p>${formatMessage(adminConfigurationMessages.encryptHint)}</p>` +
+          /******/`<div class="mb10">${formatMessage(adminConfigurationMessages.encryptHintPt1)}</div>` +
+          /******/`<div>${formatMessage(adminConfigurationMessages.encryptHintPt2)}</div>` +
+          /******/`<div class="mb10">${formatMessage(adminConfigurationMessages.encryptHintPt3)}</div>` +
+          /******/`<div>${formatMessage(adminConfigurationMessages.encryptHintPt4)}</div>` +
+          /******/`<div class="mb10">${formatMessage(adminConfigurationMessages.encryptHintPt5)}</div>` +
+          /******/`<div>${formatMessage(adminConfigurationMessages.encryptHintPt6)}</div>` +
+          /******/`<div>${formatMessage(adminConfigurationMessages.encryptHintPt7)}</div>` +
+          /******/`<div>${formatMessage(adminConfigurationMessages.encryptHintPt8)}</div>` +
+          /******/`<div>${formatMessage(adminConfigurationMessages.encryptHintPt9)}</div>` +
```

Comment:
- Remove bullets from Pts 7-9 and mark it up as a `ul`.
- Make all this markup string returned from a local function so you can use it in the two places where it is used on this screen.
- Now that the message is broken apart into pieces, you may add the bolding and other stylistic features originally not possible due to XML-like nesting and lib
- Use `p`'s instead of a bunch of divs where suitable.